src/Sulu/Bundle/CategoryBundle/Content/Types/CategorySelection.php

```diff
@@ -18,9 +18,9 @@
 use Sulu\Component\Content\ContentTypeExportInterface;
 
 /**
- * Content Type for the CategoryList, uses the CategoryManager-Service and the Datagrid from Husky.
+ * Content Type for the CategorySelection, uses the CategoryManager-Service and the Datagrid from Husky.
```

Comment:
I would remove this comment :see_no_evil: Doesn't add any real value, and the `Husky` part is also not true anymore :sweat_smile: